app/views/ui-components/v1/layouts/_footer.html.slim

```diff
@@ -1,9 +1,9 @@
 footer.footer-wrapper#footer-uic class=("#{"pre_prod_nav_color" if ENV['ENROLL_REVIEW_ENVIRONMENT'] == 'true'}")
   .container
-    .col-md-6
+    .col-md-3
```

Comment:
please revert all changes to footer and move chat widget partial to be a child of the `body` element